response_operations_ui/contexts/collection_exercise.py

```diff
@@ -27,20 +27,22 @@
 }
 
 
-def build_ce_context(ce_details: dict, has_edit_permission: bool, locked: bool) -> dict:
-    ce = ce_details["collection_exercise"]
-    events = ce_details["events"]
-    short_name = ce_details["survey"]["shortName"]
-    survey_ref = ce_details["survey"]["surveyRef"]
-    collection_instruments = ce_details["collection_instruments"]
+def build_ce_context(
+    collection_exercise, survey, collection_instruments, events, has_edit_permission: bool, locked: bool
```

Comment:
I just split out the individual components for each, i.e collection, exercise, survey, instruments, events, rather than ce_details dict